utilities/rpp-unittests/HOST_NEW/BatchPD_host_pln3.cpp

```diff
@@ -22,6 +22,27 @@ typedef half Rpp16f;
 
 #define RPPPIXELCHECK(pixel) (pixel < (Rpp32f)0) ? ((Rpp32f)0) : ((pixel < (Rpp32f)255) ? pixel : ((Rpp32f)255))
 
+void swap (unsigned int *a, unsigned int *b)
+{
+    unsigned int temp = *a;
+    *a = *b;
+    *b = temp;
+}
+
+void randomize (unsigned int arr[], unsigned int n)
+{
+    // Use a different seed value so that
```

Comment:
Please change the language without the 'we's' in a single line to something like - Use different seed values to get different results on every run